test/command/cmd.go

```
@@ -204,35 +214,53 @@ func (s *Starter) KillAll() {
 			continue
 		}
 		if v, ok := cmd.Value.(*exec.Cmd); ok {
-			s.kill(v)
+			err := s.kill(v)
```

I think we should continue even if failed to kill some cmd. And we can return the errors with a slice. WDYT?

---

use a slice?
It will change a lot of code.
or keep it as same as before, do not handle this error.WDYT

---

IMOP, that's OK. 😄